src/styled components/HeaderText.ts

```diff
@@ -0,0 +1,8 @@
+import styled from "styled-components";
```

Comment:
Do not use whatspaces like for this directory name, it might be problematic in some operating systems

Reply:
+ I'm not sure what is your purpose here, in here you are starting using styled-components, in other files you are using scss for styling, on the other hand you have mui components... why's that?